apps/client/src/app/challenges/add/page.tsx

```diff
@@ -11,24 +13,24 @@ const items = [
 const AddChallenge = () => {
   const handleAddSubmit = async (values: ChallengeFormValues) => {
     console.log(values, 'add submit values');
-    // TODO POST 
+    // TODO POST
     // If success navigate to /challenge
   };
 
   return (
     <>
-      <Box component="section" p="2em" w={'100%'} mx="auto">
+      <Container maw="1800px" p="2xl" w={'100%'}>
```

Comment:
Why is the max width `1800`? 🤔
Also, `px` is not necessary.

Reply:
Removed 1800. 
Removed w=100%